packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -171,6 +176,8 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         {...this.props}
         chartTitle={this._getChartTitle()}
         points={this._points}
+        yMinValue={this.props.supportNegativeYValues ? this._yMin : 0}
+        supportNegativeValuesForYAxis={this.props.supportNegativeYValues}
```

Comment:
rename to supportNegativeData. We will use same property when we add support for 2nd and 3rd quadrant.

Reply:
Okay

Reply:
I dont see this updated at all places